content/documentation/PinePhone_Pro/Software/Multi-distribution_image.adoc

```diff
@@ -0,0 +1,291 @@
+
+== Building
+
+Insert microSD card into the Linux computer. Find out the device name using the command `lsblk` and then make sure there are no signatures or partitions left with the command `sudo wipefs /dev/*[DEVICE]*`, where *[DEVICE]* is the name of the device displayed in the _lsblk_ command. To erase all signatures, type `sudo wipefs --all --force /dev/*[DEVICE]*`.
+
+Overwrite the microSD card with zeroes by either overwriting the first sectors with zeroes (command 1) or the whole device (command 2): 
+
+. `sudo dd if=/dev/zero of=/dev/**[DEVICE]** status=progress bs=32768 count=1`
+. `sudo dd if=/dev/zero of=/dev/**[DEVICE]** status=progress bs=32768 count=$(expr $(lsblk -bno SIZE /dev/[DEVICE] | head -1) \/ 32768)`
+
+=== Partitioning
+
+Set the _PARTSIZE_ variable to the size of your microSD card. A minimum capacity of 64 GB for the microSD card is recommended for 5 distributions.
+
+For example for a microSD card with 16 GB, set:
+
+----
+PARTSIZE=16G
+----
+
+Then partition the microSD card for the multiple distributions (make sure to replace _[DEVICE]_ with the correct device):
+
+----
+sudo sfdisk /dev/[DEVICE] --wipe always <<EOF
+ label: gpt
+ first-lba: 64
+ table-length: 8
+ attrs=RequiredPartition, type=D7B1F817-AA75-2F4F-830D-84818A145370, start=64, size=32704, name="uboot_raw"
+ attrs="RequiredPartition,LegacyBIOSBootable", size=$PARTSIZE, name="ARCH"
+ attrs="RequiredPartition,LegacyBIOSBootable", size=$PARTSIZE, name="MANJARO"
+ attrs="RequiredPartition,LegacyBIOSBootable", size=$PARTSIZE, name="MOBIAN"
+ attrs="RequiredPartition,LegacyBIOSBootable", size=$PARTSIZE, name="PMOS"
+ attrs="RequiredPartition,LegacyBIOSBootable", size=$PARTSIZE, name="UT"
+ attrs="RequiredPartition,LegacyBIOSBootable", size=+, name="extra"
+EOF
+----
+
+Expected results:
+
+----
+Checking that no-one is using this disk right now ... OK
+Disk /dev/sd[...]: 118.16 GiB, 126877696000 bytes, 247808000 sectors
+Disk model: microSD card Reader  
+Units: sectors of 1 * 512 = 512 bytes
+Sector size (logical/physical): 512 bytes / 512 bytes
+I/O size (minimum/optimal): 512 bytes / 512 bytes
+>>> Script header accepted.
+New situation:
+Disklabel type: gpt
+Disk identifier: A012E9D0-B4EB-4677-926F-D93AE4C696FA
+ Device    Start       End  Sectors   Size Type
+ sdb1         64     32767    32704   16M  unknown
+ sdb2      32768  33587199 33554432   16G  Linux fs
+ sdb3   33587200  67141631 33554432   16G  Linux fs
+ sdb4   67141632 100696063 33554432   16G  Linux fs
+ sdb5  100696064 134250495 33554432   16G  Linux fs
+ sdb6  134250496 167804927 33554432   16G  Linux fs
+ sdb7  167804928 247805951 80001024 38.1G  Linux fs
+The partition table has been altered.
+Calling ioctl() to re-read partition table.
+Syncing disks.
+----
+
+=== Install U-Boot
+
+Download the following U-Boot build (the source code of the build can be on link:https://xff.cz/git/u-boot/tree/?h=ppp-2023.07[xff.cz]):
+
+----
+cd ~/rk2aw
```

Comment:
At this stage the user may not have an `~/rk2aw` directory, since the earlier instructions got them to download `ppp.tar.gz` directly to their phone. So at a minimum it should be:

```shell
mkdir -p ~/rk2aw && cd ~/rk2aw
```

In an earlier commit I opted to call the directory `~/u-boot` instead, to avoid confusion for users that chose not to install rk2aw. Alternatively we can remove the `cd` altogether; since we are downloading and extracting the archive in the same section, it doesn't really matter what directory the user starts in.